nmap.go

```diff
@@ -117,52 +118,107 @@ func (s *Scanner) Run() (*Run, error) {
 }
 
 // RunAsync runs nmap asynchronously and returns error.
-func (s *Scanner) RunAsync() error {
+func (s *Scanner) RunAsync() (<-chan []byte, <-chan []byte, error) {
```

Comment:
WDYT of naming those return variables? This would make it more obvious what to expect, by simply glancing at the function's prototype :)

```suggestion
func (s *Scanner) RunAsync() (stdoutChan <-chan []byte, stderrChan <-chan []byte, err error) {
```